browser-extension/utils.js

```diff
@@ -47,21 +47,23 @@ export const fetchTournesolApi = async (url, method, data) => {
 }
 
 export const addRateLater = async (video_id) => {
-  const resp = await fetchTournesolApi('video/', 'POST', {video_id: video_id})
-    .then(() =>
-      fetchTournesolApi(
-        'users/me/video_rate_later/',
-        'POST',
-        {video: {video_id: video_id}}
-      )
-    );
-  if (resp && resp.ok) {
+  const addVideoResponse = await fetchTournesolApi('video/', 'POST', {video_id: video_id});
+  if(!addVideoResponse || addVideoResponse.status === 409){
```

Comment:
Hello @FabienRoger 

I managed to make the extension to work by using `401` instead of `409`. Currently the API automatically returns 401 Unauthorized when a call is made by an anonymous user.

```suggestion
  if(!addVideoResponse || addVideoResponse.status === 401) {
```

To be clear, the third-party Python package Django REST framework we used in the back end always returns a 401 or a 403 if a permission check fails.

If you want to better understand what's going on under the hood, you can check the code and its documentation.

- code  https://github.com/tournesol-app/tournesol/blob/main/backend/tournesol/views/video.py#L78
- docs  https://www.django-rest-framework.org/api-guide/permissions/#how-permissions-are-determined

Reply:
Ok, thank you for the explanation!